web/themes/custom/pineapple/sass/base/forms/_admin-feedback-form.scss

```diff
@@ -0,0 +1,13 @@
+.block-admin-feedback {
+  #feedback-message {
+    margin-top: $unit *3;
+
+    .admin-feedback-form {
+      padding: 0;
+    }
+  }
+
+  h2 {
```

Comment:
Style this h2 so that it looks like other headings on the site, you can use ```@extend``` to apply a class to another selector using sass: https://stackoverflow.com/a/9595542

Reply:
Done.